tests/components/switch/test_template.py

```diff
@@ -131,15 +132,11 @@ def test_template_syntax_error(self):
                 }
             }
         })
-
-        state = self.hass.states.set('switch.test_state', STATE_ON)
-        self.hass.pool.block_till_done()
```

Comment:
Shouldn't we at least try to set a state before reading? (Keeping lines 135 and 136?)